pkg/geo/decode.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package geo
+
+import (
+	"github.com/cockroachdb/errors"
+	"github.com/pierrre/geohash"
+)
+
+// GeohashToPoint converts a geohash to lat/long representation.
```

Comment:
nit: rename to GeoHashToPoint
I think this should be "converts a GeoHash to a Geometry Point using a Lng/Lat representation of the GeoHash".

Reply:
done